src/main/java/com/microsoftopentechnologies/windowsazurestorage/AzureStorageBlueArtifact.java

```diff
@@ -0,0 +1,69 @@
+package com.microsoftopentechnologies.windowsazurestorage;
+
+import hudson.Extension;
+import hudson.model.Run;
+import io.jenkins.blueocean.rest.Reachable;
+import io.jenkins.blueocean.rest.factory.BlueArtifactFactory;
+import io.jenkins.blueocean.rest.hal.Link;
+import io.jenkins.blueocean.rest.model.BlueArtifact;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public final class AzureStorageBlueArtifact extends BlueArtifact {
+    private final AzureBlobAction action;
+    private final AzureBlob artifact;
+
+    public AzureStorageBlueArtifact(AzureBlobAction action, AzureBlob artifact, Link parent) {
+        super(parent);
+        this.action = action;
+        this.artifact = artifact;
+    }
+
+    @Override
+    public String getName() {
+        return artifact.getBlobName();
+    }
+
+    @Override
+    public String getPath() {
+        return artifact.getBlobName();
```

Comment:
What's a `Path` of a `BlueArtifact`? I'm curious that the `Name` and `Path` are set as the same value.

Reply:
`Path` is the relative path name from artifacts root, and the table row item's name is also set as `Path`. To consist with Azure Artifacts page which each item shown name is blob name, I think we can use blob name here too.

Reply:
Got it.